tests/openwisp2/settings.py

```diff
@@ -167,6 +167,9 @@
     },
 }
 
+# Required for SAMPLE_APP as celery auto detects tasks only from INSTALLED_APPS
+CELERY_IMPORTS = ('openwisp_monitoring.device.tasks',)
```

Comment:
move this to the `if os.environ.get('SAMPLE_APP', False):` block, otherwise we won't spot problems if we mess the  autoloading of celery tasks